From f0128787fec3de9f6ee829a065952d27bd860300 Mon Sep 17 00:00:00 2001
From: Tim Schumacher <timschumi2@arcor.de>
Date: Mon, 2 Jan 2017 12:29:29 +0100
Subject: [PATCH 03/74] mirror-manifest: Added some missing repositories

Added some repositories that were missing from the original manifest.
They were also sorted in a alphabetical order to allow easy adding/removing/editing of entries without regenerating the manifest completely

Change-Id: I31e63d0889614830ab554fe69083fec084cd8c5f
---
 default.xml | 309 +++++++++++++++++++++++++++++++++++++++++-----------
 1 file changed, 245 insertions(+), 64 deletions(-)

diff --git a/default.xml b/default.xml
index 5eb42bf..34ceb41 100644
--- a/default.xml
+++ b/default.xml
@@ -8,15 +8,10 @@
            remote="github"
            sync-j="4" />
 
-  <project name="LineageOS/AR6kSDK.3.1" />
-  <project name="LineageOS/AndroidAsync" />
-  <project name="LineageOS/CMAccountWebsite" />
-  <project name="LineageOS/CMBalance" />
-  <project name="LineageOS/Superuser" />
-  <project name="LineageOS/Widgets" />
   <project name="LineageOS/android" />
   <project name="LineageOS/android_abi_cpp" />
   <project name="LineageOS/android_art" />
+  <project name="LineageOS/AndroidAsync" />
   <project name="LineageOS/android_bionic" />
   <project name="LineageOS/android_bootable_bootloader_legacy" />
   <project name="LineageOS/android_bootable_diskinstaller" />
@@ -24,6 +19,7 @@
   <project name="LineageOS/android_bootable_recovery-cm" />
   <project name="LineageOS/android_bootable_userfastboot" />
   <project name="LineageOS/android_build" />
+  <project name="LineageOS/android_build_kati" />
   <project name="LineageOS/android_cts" />
   <project name="LineageOS/android_dalvik" />
   <project name="LineageOS/android_development" />
@@ -34,46 +30,57 @@
   <project name="LineageOS/android_device_acer_t20-common" />
   <project name="LineageOS/android_device_acer_t30-common" />
   <project name="LineageOS/android_device_advent_vega" />
+  <project name="LineageOS/android_device_alcatel_idol3" />
   <project name="LineageOS/android_device_amazon_apollo" />
   <project name="LineageOS/android_device_amazon_bowser-common" />
   <project name="LineageOS/android_device_amazon_hdx-common" />
   <project name="LineageOS/android_device_amazon_jem" />
   <project name="LineageOS/android_device_amazon_omap4-common" />
   <project name="LineageOS/android_device_amazon_otter" />
-  <project name="LineageOS/android_device_amazon_otter-common" />
   <project name="LineageOS/android_device_amazon_otter2" />
+  <project name="LineageOS/android_device_amazon_otter-common" />
   <project name="LineageOS/android_device_amazon_otterx" />
   <project name="LineageOS/android_device_amazon_tate" />
   <project name="LineageOS/android_device_amazon_thor" />
-  <project name="LineageOS/android_device_asus_P024" />
-  <project name="LineageOS/android_device_asus_Z008" />
-  <project name="LineageOS/android_device_asus_Z00A" />
-  <project name="LineageOS/android_device_asus_Z00L" />
-  <project name="LineageOS/android_device_asus_Z00T" />
+  <project name="LineageOS/android_device_ark_msm8916-common" />
+  <project name="LineageOS/android_device_ark_peach" />
   <project name="LineageOS/android_device_asus_deb" />
   <project name="LineageOS/android_device_asus_flo" />
   <project name="LineageOS/android_device_asus_fugu" />
   <project name="LineageOS/android_device_asus_grouper" />
   <project name="LineageOS/android_device_asus_mofd-common" />
   <project name="LineageOS/android_device_asus_msm8916-common" />
+  <project name="LineageOS/android_device_asus_P024" />
   <project name="LineageOS/android_device_asus_tf101" />
   <project name="LineageOS/android_device_asus_tf201" />
   <project name="LineageOS/android_device_asus_tf300t" />
   <project name="LineageOS/android_device_asus_tf700t" />
   <project name="LineageOS/android_device_asus_tf701t" />
   <project name="LineageOS/android_device_asus_tilapia" />
+  <project name="LineageOS/android_device_asus_Z008" />
+  <project name="LineageOS/android_device_asus_Z00A" />
+  <project name="LineageOS/android_device_asus_Z00D" />
+  <project name="LineageOS/android_device_asus_Z00L" />
+  <project name="LineageOS/android_device_asus_Z00T" />
   <project name="LineageOS/android_device_bn_acclaim" />
   <project name="LineageOS/android_device_bn_common" />
   <project name="LineageOS/android_device_bn_encore" />
   <project name="LineageOS/android_device_bn_hummingbird" />
   <project name="LineageOS/android_device_bn_omap4-common" />
   <project name="LineageOS/android_device_bn_ovation" />
+  <project name="LineageOS/android_device_bq_gohan" />
+  <project name="LineageOS/android_device_bq_msm8916-common" />
   <project name="LineageOS/android_device_bq_paella" />
+  <project name="LineageOS/android_device_bq_piccolo" />
+  <project name="LineageOS/android_device_bq_vegetalte" />
   <project name="LineageOS/android_device_common" />
   <project name="LineageOS/android_device_commtiva_z71" />
   <project name="LineageOS/android_device_cyanogen_full" />
   <project name="LineageOS/android_device_cyanogen_msm8916-common" />
   <project name="LineageOS/android_device_cyanogen_msm8939-common" />
+  <project name="LineageOS/android_device_cyanogen_mt6735-common" />
+  <project name="LineageOS/android_device_dell_streak7" />
+  <project name="LineageOS/android_device_elocity_a7" />
   <project name="LineageOS/android_device_geeksphone_one" />
   <project name="LineageOS/android_device_geeksphone_zero" />
   <project name="LineageOS/android_device_generic" />
@@ -88,10 +95,15 @@
   <project name="LineageOS/android_device_google_accessory_arduino" />
   <project name="LineageOS/android_device_google_accessory_demokit" />
   <project name="LineageOS/android_device_google_atv" />
+  <project name="LineageOS/android_device_google_marlin" />
+  <project name="LineageOS/android_device_google_msm8952-common" />
+  <project name="LineageOS/android_device_google_sailfish" />
+  <project name="LineageOS/android_device_google_seed" />
+  <project name="LineageOS/android_device_google_shamrock" />
   <project name="LineageOS/android_device_google_sprout" />
-  <project name="LineageOS/android_device_google_sprout-common" />
   <project name="LineageOS/android_device_google_sprout4" />
   <project name="LineageOS/android_device_google_sprout8" />
+  <project name="LineageOS/android_device_google_sprout-common" />
   <project name="LineageOS/android_device_google_steelhead" />
   <project name="LineageOS/android_device_hardkernel_odroidu2" />
   <project name="LineageOS/android_device_hardkernel_samsung" />
@@ -102,14 +114,16 @@
   <project name="LineageOS/android_device_htc_a5dwg" />
   <project name="LineageOS/android_device_htc_ace" />
   <project name="LineageOS/android_device_htc_bravo" />
-  <project name="LineageOS/android_device_htc_bravo-common" />
   <project name="LineageOS/android_device_htc_bravoc" />
+  <project name="LineageOS/android_device_htc_bravo-common" />
   <project name="LineageOS/android_device_htc_buzz" />
   <project name="LineageOS/android_device_htc_click" />
   <project name="LineageOS/android_device_htc_common" />
   <project name="LineageOS/android_device_htc_desirec" />
   <project name="LineageOS/android_device_htc_dlx" />
   <project name="LineageOS/android_device_htc_dlxj" />
+  <project name="LineageOS/android_device_htc_dlxu" />
+  <project name="LineageOS/android_device_htc_dlxub1" />
   <project name="LineageOS/android_device_htc_doubleshot" />
   <project name="LineageOS/android_device_htc_dream_sapphire" />
   <project name="LineageOS/android_device_htc_endeavoru" />
@@ -133,14 +147,15 @@
   <project name="LineageOS/android_device_htc_himawl" />
   <project name="LineageOS/android_device_htc_holiday" />
   <project name="LineageOS/android_device_htc_inc" />
+  <project name="LineageOS/android_device_htc_incs" />
   <project name="LineageOS/android_device_htc_jewel" />
   <project name="LineageOS/android_device_htc_legend" />
   <project name="LineageOS/android_device_htc_leo" />
   <project name="LineageOS/android_device_htc_liberty" />
   <project name="LineageOS/android_device_htc_m4" />
   <project name="LineageOS/android_device_htc_m7" />
-  <project name="LineageOS/android_device_htc_m7-common" />
   <project name="LineageOS/android_device_htc_m7att" />
+  <project name="LineageOS/android_device_htc_m7-common" />
   <project name="LineageOS/android_device_htc_m7spr" />
   <project name="LineageOS/android_device_htc_m7tmo" />
   <project name="LineageOS/android_device_htc_m7ul" />
@@ -157,8 +172,12 @@
   <project name="LineageOS/android_device_htc_msm8974-common" />
   <project name="LineageOS/android_device_htc_passion" />
   <project name="LineageOS/android_device_htc_passion-common" />
+  <project name="LineageOS/android_device_htc_pme" />
+  <project name="LineageOS/android_device_htc_protou" />
+  <project name="LineageOS/android_device_htc_puccinilte" />
   <project name="LineageOS/android_device_htc_pyramid" />
   <project name="LineageOS/android_device_htc_ruby" />
+  <project name="LineageOS/android_device_htc_runnymede" />
   <project name="LineageOS/android_device_htc_s4-common" />
   <project name="LineageOS/android_device_htc_saga" />
   <project name="LineageOS/android_device_htc_shooter" />
@@ -170,24 +189,28 @@
   <project name="LineageOS/android_device_htc_t6spr" />
   <project name="LineageOS/android_device_htc_t6vzw" />
   <project name="LineageOS/android_device_htc_tegra3-common" />
+  <project name="LineageOS/android_device_htc_vigor" />
   <project name="LineageOS/android_device_htc_ville" />
   <project name="LineageOS/android_device_htc_vision" />
   <project name="LineageOS/android_device_htc_vivo" />
   <project name="LineageOS/android_device_htc_vivow" />
   <project name="LineageOS/android_device_htc_zara" />
-  <project name="LineageOS/android_device_htc_zara-common" />
   <project name="LineageOS/android_device_htc_zaracl" />
+  <project name="LineageOS/android_device_htc_zara-common" />
   <project name="LineageOS/android_device_huawei_angler" />
+  <project name="LineageOS/android_device_huawei_ascend" />
   <project name="LineageOS/android_device_huawei_che10" />
   <project name="LineageOS/android_device_huawei_cherry" />
   <project name="LineageOS/android_device_huawei_g620_a2" />
   <project name="LineageOS/android_device_huawei_kiwi" />
+  <project name="LineageOS/android_device_huawei_m660" />
   <project name="LineageOS/android_device_huawei_msm8916-common" />
   <project name="LineageOS/android_device_huawei_mt2" />
   <project name="LineageOS/android_device_huawei_u8150" />
   <project name="LineageOS/android_device_huawei_u8160" />
   <project name="LineageOS/android_device_huawei_u8220" />
   <project name="LineageOS/android_device_intel_mixins" />
+  <project name="LineageOS/android_device_leeco_s2" />
   <project name="LineageOS/android_device_letv_x1na" />
   <project name="LineageOS/android_device_lge_bullhead" />
   <project name="LineageOS/android_device_lge_c660" />
@@ -200,8 +223,10 @@
   <project name="LineageOS/android_device_lge_d851" />
   <project name="LineageOS/android_device_lge_d852" />
   <project name="LineageOS/android_device_lge_d855" />
+  <project name="LineageOS/android_device_lge_d950" />
   <project name="LineageOS/android_device_lge_dory" />
   <project name="LineageOS/android_device_lge_e400" />
+  <project name="LineageOS/android_device_lge_e430" />
   <project name="LineageOS/android_device_lge_e510" />
   <project name="LineageOS/android_device_lge_e610" />
   <project name="LineageOS/android_device_lge_e720" />
@@ -216,23 +241,32 @@
   <project name="LineageOS/android_device_lge_g2m" />
   <project name="LineageOS/android_device_lge_g3-common" />
   <project name="LineageOS/android_device_lge_g4-common" />
+  <project name="LineageOS/android_device_lge_g5-common" />
   <project name="LineageOS/android_device_lge_gproj-common" />
   <project name="LineageOS/android_device_lge_h811" />
   <project name="LineageOS/android_device_lge_h815" />
+  <project name="LineageOS/android_device_lge_h830" />
+  <project name="LineageOS/android_device_lge_h850" />
   <project name="LineageOS/android_device_lge_h901" />
+  <project name="LineageOS/android_device_lge_h918" />
   <project name="LineageOS/android_device_lge_hammerhead" />
   <project name="LineageOS/android_device_lge_hammerheadcaf" />
   <project name="LineageOS/android_device_lge_iprj-common" />
   <project name="LineageOS/android_device_lge_jag3gds" />
   <project name="LineageOS/android_device_lge_jagnm" />
   <project name="LineageOS/android_device_lge_l01f" />
+  <project name="LineageOS/android_device_lge_l1m" />
+  <project name="LineageOS/android_device_lge_lgl55c" />
   <project name="LineageOS/android_device_lge_ls970" />
   <project name="LineageOS/android_device_lge_ls970-common" />
   <project name="LineageOS/android_device_lge_ls980" />
   <project name="LineageOS/android_device_lge_ls990" />
   <project name="LineageOS/android_device_lge_mako" />
+  <project name="LineageOS/android_device_lge_ms695" />
+  <project name="LineageOS/android_device_lge_ms910" />
   <project name="LineageOS/android_device_lge_msm7x27a-common" />
   <project name="LineageOS/android_device_lge_msm8226-common" />
+  <project name="LineageOS/android_device_lge_msm8996-common" />
   <project name="LineageOS/android_device_lge_p350" />
   <project name="LineageOS/android_device_lge_p500" />
   <project name="LineageOS/android_device_lge_p700" />
@@ -240,36 +274,45 @@
   <project name="LineageOS/android_device_lge_p760" />
   <project name="LineageOS/android_device_lge_p880" />
   <project name="LineageOS/android_device_lge_p920" />
+  <project name="LineageOS/android_device_lge_p925" />
   <project name="LineageOS/android_device_lge_p930" />
   <project name="LineageOS/android_device_lge_p970" />
   <project name="LineageOS/android_device_lge_p990" />
   <project name="LineageOS/android_device_lge_p999" />
   <project name="LineageOS/android_device_lge_star-common" />
   <project name="LineageOS/android_device_lge_su640" />
+  <project name="LineageOS/android_device_lge_thunderc" />
   <project name="LineageOS/android_device_lge_thunderg" />
+  <project name="LineageOS/android_device_lge_us996" />
+  <project name="LineageOS/android_device_lge_us997" />
   <project name="LineageOS/android_device_lge_v10-common" />
+  <project name="LineageOS/android_device_lge_v20-common" />
   <project name="LineageOS/android_device_lge_v400" />
   <project name="LineageOS/android_device_lge_v410" />
+  <project name="LineageOS/android_device_lge_v480" />
+  <project name="LineageOS/android_device_lge_v490" />
   <project name="LineageOS/android_device_lge_v4xx-common" />
   <project name="LineageOS/android_device_lge_v500" />
   <project name="LineageOS/android_device_lge_v909" />
   <project name="LineageOS/android_device_lge_victor-common" />
+  <project name="LineageOS/android_device_lge_vs840" />
+  <project name="LineageOS/android_device_lge_vs910" />
   <project name="LineageOS/android_device_lge_vs920" />
   <project name="LineageOS/android_device_lge_vs980" />
   <project name="LineageOS/android_device_lge_vs985" />
   <project name="LineageOS/android_device_lge_w5" />
   <project name="LineageOS/android_device_lge_w7" />
   <project name="LineageOS/android_device_lge_zee" />
-  <project name="LineageOS/android_device_malata_smb-common" />
   <project name="LineageOS/android_device_malata_smb_a1002" />
   <project name="LineageOS/android_device_malata_smb_a1004" />
   <project name="LineageOS/android_device_malata_smb_a1011" />
   <project name="LineageOS/android_device_malata_smb_b9701" />
+  <project name="LineageOS/android_device_malata_smb-common" />
   <project name="LineageOS/android_device_moto_common" />
   <project name="LineageOS/android_device_moto_everest" />
-  <project name="LineageOS/android_device_moto_shamu" />
-  <project name="LineageOS/android_device_moto_stingray" />
-  <project name="LineageOS/android_device_moto_wingray" />
+  <project name="LineageOS/android_device_motorola_asanti" />
+  <project name="LineageOS/android_device_motorola_athene" />
+  <project name="LineageOS/android_device_motorola_backflip" />
   <project name="LineageOS/android_device_motorola_clark" />
   <project name="LineageOS/android_device_motorola_common" />
   <project name="LineageOS/android_device_motorola_condor" />
@@ -278,6 +321,7 @@
   <project name="LineageOS/android_device_motorola_edison" />
   <project name="LineageOS/android_device_motorola_falcon" />
   <project name="LineageOS/android_device_motorola_ghost" />
+  <project name="LineageOS/android_device_motorola_harpia" />
   <project name="LineageOS/android_device_motorola_jordan" />
   <project name="LineageOS/android_device_motorola_jordan-common" />
   <project name="LineageOS/android_device_motorola_jordan_plus" />
@@ -286,15 +330,15 @@
   <project name="LineageOS/android_device_motorola_mb886" />
   <project name="LineageOS/android_device_motorola_morrison" />
   <project name="LineageOS/android_device_motorola_moto_msm8960" />
-  <project name="LineageOS/android_device_motorola_moto_msm8960_jbbl" />
   <project name="LineageOS/android_device_motorola_moto_msm8960dt" />
+  <project name="LineageOS/android_device_motorola_moto_msm8960_jbbl" />
   <project name="LineageOS/android_device_motorola_motus" />
   <project name="LineageOS/android_device_motorola_msm8226-common" />
   <project name="LineageOS/android_device_motorola_msm8610-common" />
   <project name="LineageOS/android_device_motorola_msm8916-common" />
   <project name="LineageOS/android_device_motorola_msm8960-common" />
-  <project name="LineageOS/android_device_motorola_msm8960_jbbl-common" />
   <project name="LineageOS/android_device_motorola_msm8960dt-common" />
+  <project name="LineageOS/android_device_motorola_msm8960_jbbl-common" />
   <project name="LineageOS/android_device_motorola_obake" />
   <project name="LineageOS/android_device_motorola_olympus" />
   <project name="LineageOS/android_device_motorola_omap4-common" />
@@ -312,7 +356,9 @@
   <project name="LineageOS/android_device_motorola_targa" />
   <project name="LineageOS/android_device_motorola_thea" />
   <project name="LineageOS/android_device_motorola_titan" />
+  <project name="LineageOS/android_device_motorola_triumph" />
   <project name="LineageOS/android_device_motorola_umts_spyder" />
+  <project name="LineageOS/android_device_motorola_vanquish-common" />
   <project name="LineageOS/android_device_motorola_victara" />
   <project name="LineageOS/android_device_motorola_xt1053" />
   <project name="LineageOS/android_device_motorola_xt1058" />
@@ -324,12 +370,17 @@
   <project name="LineageOS/android_device_motorola_xt925_jbbl" />
   <project name="LineageOS/android_device_motorola_xt926" />
   <project name="LineageOS/android_device_motorola_zeppelin" />
+  <project name="LineageOS/android_device_moto_shamu" />
+  <project name="LineageOS/android_device_moto_stingray" />
+  <project name="LineageOS/android_device_moto_wingray" />
   <project name="LineageOS/android_device_nextbit_ether" />
+  <project name="LineageOS/android_device_nubia_nx531j" />
   <project name="LineageOS/android_device_nvidia_foster" />
   <project name="LineageOS/android_device_nvidia_roth" />
   <project name="LineageOS/android_device_nvidia_shield-common" />
   <project name="LineageOS/android_device_nvidia_shieldtablet" />
   <project name="LineageOS/android_device_oneplus_bacon" />
+  <project name="LineageOS/android_device_oneplus_common" />
   <project name="LineageOS/android_device_oneplus_oneplus2" />
   <project name="LineageOS/android_device_oneplus_oneplus3" />
   <project name="LineageOS/android_device_oneplus_onyx" />
@@ -355,6 +406,7 @@
   <project name="LineageOS/android_device_qcom_msm8996" />
   <project name="LineageOS/android_device_qcom_sepolicy" />
   <project name="LineageOS/android_device_sample" />
+  <project name="LineageOS/android_device_samsung_amazing3gcri" />
   <project name="LineageOS/android_device_samsung_apexqtmo" />
   <project name="LineageOS/android_device_samsung_aries-common" />
   <project name="LineageOS/android_device_samsung_bmlunlock" />
@@ -363,12 +415,14 @@
   <project name="LineageOS/android_device_samsung_celox-common" />
   <project name="LineageOS/android_device_samsung_chagall-common" />
   <project name="LineageOS/android_device_samsung_chagalllte" />
+  <project name="LineageOS/android_device_samsung_chagallltetmo" />
   <project name="LineageOS/android_device_samsung_chagallwifi" />
+  <project name="LineageOS/android_device_samsung_charge" />
   <project name="LineageOS/android_device_samsung_cooper" />
   <project name="LineageOS/android_device_samsung_crespo" />
   <project name="LineageOS/android_device_samsung_crespo4g" />
-  <project name="LineageOS/android_device_samsung_d2-common" />
   <project name="LineageOS/android_device_samsung_d2att" />
+  <project name="LineageOS/android_device_samsung_d2-common" />
   <project name="LineageOS/android_device_samsung_d2cri" />
   <project name="LineageOS/android_device_samsung_d2lte" />
   <project name="LineageOS/android_device_samsung_d2mtr" />
@@ -377,27 +431,38 @@
   <project name="LineageOS/android_device_samsung_d2usc" />
   <project name="LineageOS/android_device_samsung_d2vzw" />
   <project name="LineageOS/android_device_samsung_d710" />
+  <project name="LineageOS/android_device_samsung_epic4g" />
   <project name="LineageOS/android_device_samsung_epicmtd" />
-  <project name="LineageOS/android_device_samsung_espresso-common" />
   <project name="LineageOS/android_device_samsung_espresso3g" />
+  <project name="LineageOS/android_device_samsung_espresso-common" />
   <project name="LineageOS/android_device_samsung_espressowifi" />
   <project name="LineageOS/android_device_samsung_exhilarate" />
   <project name="LineageOS/android_device_samsung_expressatt" />
   <project name="LineageOS/android_device_samsung_exynos5420-common" />
+  <project name="LineageOS/android_device_samsung_fascinate" />
   <project name="LineageOS/android_device_samsung_fascinatemtd" />
   <project name="LineageOS/android_device_samsung_galaxy5" />
+  <project name="LineageOS/android_device_samsung_galaxyace" />
   <project name="LineageOS/android_device_samsung_galaxys2" />
-  <project name="LineageOS/android_device_samsung_galaxys2-common" />
   <project name="LineageOS/android_device_samsung_galaxys2att" />
+  <project name="LineageOS/android_device_samsung_galaxys2-common" />
+  <project name="LineageOS/android_device_samsung_galaxys4g" />
   <project name="LineageOS/android_device_samsung_galaxysbmtd" />
   <project name="LineageOS/android_device_samsung_galaxysmtd" />
-  <project name="LineageOS/android_device_samsung_gts2-common" />
+  <project name="LineageOS/android_device_samsung_golden" />
   <project name="LineageOS/android_device_samsung_gts210ltexx" />
   <project name="LineageOS/android_device_samsung_gts210wifi" />
+  <project name="LineageOS/android_device_samsung_gts2-common" />
   <project name="LineageOS/android_device_samsung_hercules" />
+  <project name="LineageOS/android_device_samsung_hero2lte" />
+  <project name="LineageOS/android_device_samsung_hero-common" />
+  <project name="LineageOS/android_device_samsung_herolte" />
+  <project name="LineageOS/android_device_samsung_herring" />
+  <project name="LineageOS/android_device_samsung_hl3g" />
+  <project name="LineageOS/android_device_samsung_hllte" />
   <project name="LineageOS/android_device_samsung_hlte" />
-  <project name="LineageOS/android_device_samsung_hlte-common" />
   <project name="LineageOS/android_device_samsung_hltecan" />
+  <project name="LineageOS/android_device_samsung_hlte-common" />
   <project name="LineageOS/android_device_samsung_hltespr" />
   <project name="LineageOS/android_device_samsung_hltetmo" />
   <project name="LineageOS/android_device_samsung_hlteusc" />
@@ -410,6 +475,7 @@
   <project name="LineageOS/android_device_samsung_i9100g" />
   <project name="LineageOS/android_device_samsung_i9103" />
   <project name="LineageOS/android_device_samsung_i925" />
+  <project name="LineageOS/android_device_samsung_i927" />
   <project name="LineageOS/android_device_samsung_i9300" />
   <project name="LineageOS/android_device_samsung_i9305" />
   <project name="LineageOS/android_device_samsung_i9500" />
@@ -425,21 +491,26 @@
   <project name="LineageOS/android_device_samsung_jflteusc" />
   <project name="LineageOS/android_device_samsung_jfltevzw" />
   <project name="LineageOS/android_device_samsung_jfltexx" />
+  <project name="LineageOS/android_device_samsung_k3gxx" />
   <project name="LineageOS/android_device_samsung_klimt-common" />
   <project name="LineageOS/android_device_samsung_klimtlte" />
   <project name="LineageOS/android_device_samsung_klimtwifi" />
   <project name="LineageOS/android_device_samsung_klte" />
-  <project name="LineageOS/android_device_samsung_klte-common" />
   <project name="LineageOS/android_device_samsung_kltechn" />
   <project name="LineageOS/android_device_samsung_kltechnduo" />
+  <project name="LineageOS/android_device_samsung_klte-common" />
   <project name="LineageOS/android_device_samsung_klteduos" />
   <project name="LineageOS/android_device_samsung_kltedv" />
   <project name="LineageOS/android_device_samsung_kltekdi" />
+  <project name="LineageOS/android_device_samsung_kltekor" />
   <project name="LineageOS/android_device_samsung_kltespr" />
   <project name="LineageOS/android_device_samsung_klteusc" />
   <project name="LineageOS/android_device_samsung_kltevzw" />
   <project name="LineageOS/android_device_samsung_kona-common" />
   <project name="LineageOS/android_device_samsung_ks01lte" />
+  <project name="LineageOS/android_device_samsung_kylepro" />
+  <project name="LineageOS/android_device_samsung_kyleprods" />
+  <project name="LineageOS/android_device_samsung_kyleve" />
   <project name="LineageOS/android_device_samsung_l900" />
   <project name="LineageOS/android_device_samsung_lt03lte" />
   <project name="LineageOS/android_device_samsung_lt03wifi" />
@@ -447,11 +518,13 @@
   <project name="LineageOS/android_device_samsung_manta" />
   <project name="LineageOS/android_device_samsung_mesmerizemtd" />
   <project name="LineageOS/android_device_samsung_mondrianwifi" />
+  <project name="LineageOS/android_device_samsung_ms013g" />
   <project name="LineageOS/android_device_samsung_msm8226-common" />
   <project name="LineageOS/android_device_samsung_msm8660-common" />
   <project name="LineageOS/android_device_samsung_msm8930-common" />
   <project name="LineageOS/android_device_samsung_msm8960-common" />
   <project name="LineageOS/android_device_samsung_msm8974-common" />
+  <project name="LineageOS/android_device_samsung_n1awifi" />
   <project name="LineageOS/android_device_samsung_n5100" />
   <project name="LineageOS/android_device_samsung_n5110" />
   <project name="LineageOS/android_device_samsung_n5120" />
@@ -464,8 +537,8 @@
   <project name="LineageOS/android_device_samsung_nobleltetmo" />
   <project name="LineageOS/android_device_samsung_omap4-common" />
   <project name="LineageOS/android_device_samsung_p1" />
-  <project name="LineageOS/android_device_samsung_p1-common" />
   <project name="LineageOS/android_device_samsung_p1c" />
+  <project name="LineageOS/android_device_samsung_p1-common" />
   <project name="LineageOS/android_device_samsung_p1l" />
   <project name="LineageOS/android_device_samsung_p1n" />
   <project name="LineageOS/android_device_samsung_p3" />
@@ -479,34 +552,41 @@
   <project name="LineageOS/android_device_samsung_p4vzw" />
   <project name="LineageOS/android_device_samsung_p4wifi" />
   <project name="LineageOS/android_device_samsung_p5" />
-  <project name="LineageOS/android_device_samsung_p5-common" />
   <project name="LineageOS/android_device_samsung_p5100" />
   <project name="LineageOS/android_device_samsung_p5110" />
   <project name="LineageOS/android_device_samsung_p5113" />
   <project name="LineageOS/android_device_samsung_p5att" />
+  <project name="LineageOS/android_device_samsung_p5-common" />
   <project name="LineageOS/android_device_samsung_p5wifi" />
+  <project name="LineageOS/android_device_samsung_p7100" />
   <project name="LineageOS/android_device_samsung_picassowifi" />
   <project name="LineageOS/android_device_samsung_piranha" />
+  <project name="LineageOS/android_device_samsung_prevail2spr" />
   <project name="LineageOS/android_device_samsung_qcom-common" />
-  <project name="LineageOS/android_device_samsung_quincy-common" />
   <project name="LineageOS/android_device_samsung_quincyatt" />
+  <project name="LineageOS/android_device_samsung_quincy-common" />
   <project name="LineageOS/android_device_samsung_quincytmo" />
   <project name="LineageOS/android_device_samsung_r950" />
   <project name="LineageOS/android_device_samsung_s3ve3g" />
   <project name="LineageOS/android_device_samsung_s5670" />
   <project name="LineageOS/android_device_samsung_sc02c" />
-  <project name="LineageOS/android_device_samsung_serrano-common" />
   <project name="LineageOS/android_device_samsung_serrano3gxx" />
+  <project name="LineageOS/android_device_samsung_serrano-common" />
+  <project name="LineageOS/android_device_samsung_serranodsdd" />
   <project name="LineageOS/android_device_samsung_serranoltexx" />
   <project name="LineageOS/android_device_samsung_showcasemtd" />
+  <project name="LineageOS/android_device_samsung_sidekick4g" />
   <project name="LineageOS/android_device_samsung_skyrocket" />
   <project name="LineageOS/android_device_samsung_smdk4412-common" />
   <project name="LineageOS/android_device_samsung_smdk4412-qcom-common" />
+  <project name="LineageOS/android_device_samsung_smdk5260-common" />
   <project name="LineageOS/android_device_samsung_superior" />
   <project name="LineageOS/android_device_samsung_t0lte" />
   <project name="LineageOS/android_device_samsung_t0lteatt" />
   <project name="LineageOS/android_device_samsung_t0ltecdma" />
+  <project name="LineageOS/android_device_samsung_t0ltespr" />
   <project name="LineageOS/android_device_samsung_t0ltetmo" />
+  <project name="LineageOS/android_device_samsung_t0ltevzw" />
   <project name="LineageOS/android_device_samsung_t769" />
   <project name="LineageOS/android_device_samsung_tass" />
   <project name="LineageOS/android_device_samsung_toro" />
@@ -521,11 +601,13 @@
   <project name="LineageOS/android_device_samsung_trlteusc" />
   <project name="LineageOS/android_device_samsung_trltexx" />
   <project name="LineageOS/android_device_samsung_tuna" />
+  <project name="LineageOS/android_device_samsung_v1awifi" />
   <project name="LineageOS/android_device_samsung_v2wifixx" />
   <project name="LineageOS/android_device_samsung_vibrant" />
   <project name="LineageOS/android_device_samsung_vibrantmtd" />
   <project name="LineageOS/android_device_samsung_vzwtab" />
   <project name="LineageOS/android_device_samsung_ypg1" />
+  <project name="LineageOS/android_device_samsung_zenlte" />
   <project name="LineageOS/android_device_samsung_zero-common" />
   <project name="LineageOS/android_device_samsung_zerofltexx" />
   <project name="LineageOS/android_device_semc_anzu" />
@@ -546,8 +628,8 @@
   <project name="LineageOS/android_device_semc_smultron" />
   <project name="LineageOS/android_device_semc_urushi" />
   <project name="LineageOS/android_device_semc_zeus" />
-  <project name="LineageOS/android_device_semc_zeus-common" />
   <project name="LineageOS/android_device_semc_zeusc" />
+  <project name="LineageOS/android_device_semc_zeus-common" />
   <project name="LineageOS/android_device_smartfren_rendang" />
   <project name="LineageOS/android_device_sony_amami" />
   <project name="LineageOS/android_device_sony_aoba" />
@@ -564,7 +646,11 @@
   <project name="LineageOS/android_device_sony_hikari" />
   <project name="LineageOS/android_device_sony_honami" />
   <project name="LineageOS/android_device_sony_huashan" />
+  <project name="LineageOS/android_device_sony_ivy" />
   <project name="LineageOS/android_device_sony_jlo" />
+  <project name="LineageOS/android_device_sony_karin" />
+  <project name="LineageOS/android_device_sony_karin_windy" />
+  <project name="LineageOS/android_device_sony_kitakami-common" />
   <project name="LineageOS/android_device_sony_kumquat" />
   <project name="LineageOS/android_device_sony_lotus" />
   <project name="LineageOS/android_device_sony_mesona" />
@@ -589,6 +675,8 @@
   <project name="LineageOS/android_device_sony_seagull" />
   <project name="LineageOS/android_device_sony_shinano-common" />
   <project name="LineageOS/android_device_sony_sirius" />
+  <project name="LineageOS/android_device_sony_sumire" />
+  <project name="LineageOS/android_device_sony_suzuran" />
   <project name="LineageOS/android_device_sony_tamsui-common" />
   <project name="LineageOS/android_device_sony_tamsui-common-legacy" />
   <project name="LineageOS/android_device_sony_taoshan" />
@@ -607,22 +695,33 @@
   <project name="LineageOS/android_device_toshiba_betelgeuse" />
   <project name="LineageOS/android_device_wileyfox_crackling" />
   <project name="LineageOS/android_device_wileyfox_kipper" />
+  <project name="LineageOS/android_device_wileyfox_porridge" />
+  <project name="LineageOS/android_device_wileyfox_porridgek3" />
   <project name="LineageOS/android_device_wingtech_wt88047" />
   <project name="LineageOS/android_device_xiaomi_aries" />
+  <project name="LineageOS/android_device_xiaomi_armani" />
   <project name="LineageOS/android_device_xiaomi_cancro" />
+  <project name="LineageOS/android_device_xiaomi_ferrari" />
+  <project name="LineageOS/android_device_xiaomi_gemini" />
+  <project name="LineageOS/android_device_xiaomi_hydrogen" />
+  <project name="LineageOS/android_device_xiaomi_ido" />
   <project name="LineageOS/android_device_xiaomi_kenzo" />
+  <project name="LineageOS/android_device_xiaomi_msm8956-common" />
+  <project name="LineageOS/android_device_yuhua_x2" />
   <project name="LineageOS/android_device_yu_jalebi" />
   <project name="LineageOS/android_device_yu_lettuce" />
   <project name="LineageOS/android_device_yu_sambar" />
   <project name="LineageOS/android_device_yu_tomato" />
-  <project name="LineageOS/android_device_yuhua_x2" />
+  <project name="LineageOS/android_device_zte_axon7" />
   <project name="LineageOS/android_device_zte_blade" />
   <project name="LineageOS/android_device_zte_nx510j" />
+  <project name="LineageOS/android_device_zte_skate" />
+  <project name="LineageOS/android_device_zte_V8000" />
   <project name="LineageOS/android_device_zte_v9" />
+  <project name="LineageOS/android_device_zte_warp2" />
+  <project name="LineageOS/android_device_zte_x500" />
   <project name="LineageOS/android_device_zuk_ham" />
   <project name="LineageOS/android_docs_source.android.com" />
-  <project name="LineageOS/android_external_Focal" />
-  <project name="LineageOS/android_external_JakeWharton_butterknife" />
   <project name="LineageOS/android_external_aac" />
   <project name="LineageOS/android_external_ahbottomnavigation" />
   <project name="LineageOS/android_external_alsa-lib" />
@@ -631,10 +730,10 @@
   <project name="LineageOS/android_external_android-mock" />
   <project name="LineageOS/android_external_android-visualizer" />
   <project name="LineageOS/android_external_ant-glob" />
+  <project name="LineageOS/android_external_antlr" />
   <project name="LineageOS/android_external_ant-wireless_ant_native" />
-  <project name="LineageOS/android_external_ant-wireless_ant_service" />
   <project name="LineageOS/android_external_ant-wireless_antradio-library" />
-  <project name="LineageOS/android_external_antlr" />
+  <project name="LineageOS/android_external_ant-wireless_ant_service" />
   <project name="LineageOS/android_external_apache-harmony" />
   <project name="LineageOS/android_external_apache-http" />
   <project name="LineageOS/android_external_apache-qp" />
@@ -663,8 +762,6 @@
   <project name="LineageOS/android_external_cardslib" />
   <project name="LineageOS/android_external_checkpolicy" />
   <project name="LineageOS/android_external_chromium" />
-  <project name="LineageOS/android_external_chromium-trace" />
-  <project name="LineageOS/android_external_chromium-webview" />
   <project name="LineageOS/android_external_chromium_org" />
   <project name="LineageOS/android_external_chromium_org_third_party_boringssl_src" />
   <project name="LineageOS/android_external_chromium_org_third_party_brotli_src" />
@@ -680,6 +777,8 @@
   <project name="LineageOS/android_external_chromium_org_third_party_usrsctp_usrsctplib" />
   <project name="LineageOS/android_external_chromium_org_third_party_webrtc" />
   <project name="LineageOS/android_external_chromium_org_v8" />
+  <project name="LineageOS/android_external_chromium-trace" />
+  <project name="LineageOS/android_external_chromium-webview" />
   <project name="LineageOS/android_external_cibu-fonts" />
   <project name="LineageOS/android_external_clang" />
   <project name="LineageOS/android_external_clearsilver" />
@@ -696,6 +795,7 @@
   <project name="LineageOS/android_external_deqp" />
   <project name="LineageOS/android_external_dexmaker" />
   <project name="LineageOS/android_external_dhcpcd" />
+  <project name="LineageOS/android_external_dhcpcd-6.8.2" />
   <project name="LineageOS/android_external_dnscrypt_dnscrypt-proxy" />
   <project name="LineageOS/android_external_dnscrypt_libsodium" />
   <project name="LineageOS/android_external_dnsmasq" />
@@ -726,14 +826,15 @@
   <project name="LineageOS/android_external_ffmpeg" />
   <project name="LineageOS/android_external_fio" />
   <project name="LineageOS/android_external_flac" />
+  <project name="LineageOS/android_external_Focal" />
   <project name="LineageOS/android_external_fonttools" />
   <project name="LineageOS/android_external_freetype" />
   <project name="LineageOS/android_external_fsck_msdos" />
   <project name="LineageOS/android_external_fuse" />
   <project name="LineageOS/android_external_ganymed-ssh2" />
   <project name="LineageOS/android_external_gcc-demangle" />
-  <project name="LineageOS/android_external_gello-env" />
   <project name="LineageOS/android_external_gello_build" />
+  <project name="LineageOS/android_external_gello-env" />
   <project name="LineageOS/android_external_genext2fs" />
   <project name="LineageOS/android_external_giflib" />
   <project name="LineageOS/android_external_glide" />
@@ -764,6 +865,7 @@
   <project name="LineageOS/android_external_ipsec-tools" />
   <project name="LineageOS/android_external_iptables" />
   <project name="LineageOS/android_external_jack" />
+  <project name="LineageOS/android_external_JakeWharton_butterknife" />
   <project name="LineageOS/android_external_jarjar" />
   <project name="LineageOS/android_external_javasqlite" />
   <project name="LineageOS/android_external_javassist" />
@@ -932,12 +1034,12 @@
   <project name="LineageOS/android_external_webkit" />
   <project name="LineageOS/android_external_webp" />
   <project name="LineageOS/android_external_webrtc" />
-  <project name="LineageOS/android_external_whispersystems_TextSecure" />
-  <project name="LineageOS/android_external_whispersystems_WhisperPush" />
   <project name="LineageOS/android_external_whispersystems_curve25519-java" />
   <project name="LineageOS/android_external_whispersystems_libaxolotl-android" />
   <project name="LineageOS/android_external_whispersystems_libtextsecure-java" />
   <project name="LineageOS/android_external_whispersystems_libwhisperpush" />
+  <project name="LineageOS/android_external_whispersystems_TextSecure" />
+  <project name="LineageOS/android_external_whispersystems_WhisperPush" />
   <project name="LineageOS/android_external_wireless-tools" />
   <project name="LineageOS/android_external_wpa_supplicant" />
   <project name="LineageOS/android_external_wpa_supplicant_6" />
@@ -1013,8 +1115,8 @@
   <project name="LineageOS/android_hardware_cm" />
   <project name="LineageOS/android_hardware_cyanogen" />
   <project name="LineageOS/android_hardware_gpsshim" />
-  <project name="LineageOS/android_hardware_intel_audio_media" />
   <project name="LineageOS/android_hardware_intel_audiocomms_utilities" />
+  <project name="LineageOS/android_hardware_intel_audio_media" />
   <project name="LineageOS/android_hardware_intel_bootstub" />
   <project name="LineageOS/android_hardware_intel_common_bd_prov" />
   <project name="LineageOS/android_hardware_intel_common_libmix" />
@@ -1042,6 +1144,7 @@
   <project name="LineageOS/android_hardware_qcom_audio" />
   <project name="LineageOS/android_hardware_qcom_audio-bfam" />
   <project name="LineageOS/android_hardware_qcom_audio-caf" />
+  <project name="LineageOS/android_hardware_qcom_bootctrl" />
   <project name="LineageOS/android_hardware_qcom_bt" />
   <project name="LineageOS/android_hardware_qcom_camera" />
   <project name="LineageOS/android_hardware_qcom_display" />
@@ -1068,19 +1171,22 @@
   <project name="LineageOS/android_hardware_samsung" />
   <project name="LineageOS/android_hardware_samsung_slsi-cm_exynos" />
   <project name="LineageOS/android_hardware_samsung_slsi-cm_exynos5" />
+  <project name="LineageOS/android_hardware_samsung_slsi-cm_exynos5260" />
   <project name="LineageOS/android_hardware_samsung_slsi-cm_exynos5410" />
   <project name="LineageOS/android_hardware_samsung_slsi-cm_exynos5420" />
+  <project name="LineageOS/android_hardware_samsung_slsi-cm_exynos5422" />
   <project name="LineageOS/android_hardware_samsung_slsi-cm_exynos5433" />
   <project name="LineageOS/android_hardware_samsung_slsi-cm_exynos7420" />
+  <project name="LineageOS/android_hardware_samsung_slsi-cm_exynos8890" />
   <project name="LineageOS/android_hardware_samsung_slsi-cm_openmax" />
   <project name="LineageOS/android_hardware_samsung_slsi_exynos" />
   <project name="LineageOS/android_hardware_samsung_slsi_exynos5" />
-  <project name="LineageOS/android_hardware_samsung_slsi_exynos5-insignal" />
   <project name="LineageOS/android_hardware_samsung_slsi_exynos5410" />
   <project name="LineageOS/android_hardware_samsung_slsi_exynos5420" />
+  <project name="LineageOS/android_hardware_samsung_slsi_exynos5-insignal" />
   <project name="LineageOS/android_hardware_samsung_slsi_openmax" />
-  <project name="LineageOS/android_hardware_sony_DASH" />
   <project name="LineageOS/android_hardware_sony_camera" />
+  <project name="LineageOS/android_hardware_sony_DASH" />
   <project name="LineageOS/android_hardware_sony_macaddrsetup" />
   <project name="LineageOS/android_hardware_sony_thermanager" />
   <project name="LineageOS/android_hardware_sony_timekeep" />
@@ -1093,6 +1199,7 @@
   <project name="LineageOS/android_hardware_ti_omx" />
   <project name="LineageOS/android_hardware_ti_syslink" />
   <project name="LineageOS/android_hardware_ti_tiler" />
+  <project name="LineageOS/android_hardware_ti_wl12xx" />
   <project name="LineageOS/android_hardware_ti_wlan" />
   <project name="LineageOS/android_hardware_ti_wpan" />
   <project name="LineageOS/android_kernel_acer_t20-common" />
@@ -1101,25 +1208,33 @@
   <project name="LineageOS/android_kernel_amazon_bowser-common" />
   <project name="LineageOS/android_kernel_amazon_hdx-common" />
   <project name="LineageOS/android_kernel_amazon_otter-common" />
-  <project name="LineageOS/android_kernel_asus_P024" />
+  <project name="LineageOS/android_kernel_ark_msm8916" />
   <project name="LineageOS/android_kernel_asus_fugu" />
   <project name="LineageOS/android_kernel_asus_grouper" />
   <project name="LineageOS/android_kernel_asus_moorefield" />
   <project name="LineageOS/android_kernel_asus_msm8916" />
+  <project name="LineageOS/android_kernel_asus_P024" />
   <project name="LineageOS/android_kernel_asus_tf101" />
   <project name="LineageOS/android_kernel_asus_tf201" />
   <project name="LineageOS/android_kernel_asus_tf300t" />
   <project name="LineageOS/android_kernel_asus_tf700t" />
   <project name="LineageOS/android_kernel_asus_tf701t" />
+  <project name="LineageOS/android_kernel_asus_Z00D" />
   <project name="LineageOS/android_kernel_bn_acclaim" />
   <project name="LineageOS/android_kernel_bn_encore" />
   <project name="LineageOS/android_kernel_bn_omap" />
   <project name="LineageOS/android_kernel_bq_msm8916" />
+  <project name="LineageOS/android_kernel_bq_msm8939" />
+  <project name="LineageOS/android_kernel_bq_msm8976" />
   <project name="LineageOS/android_kernel_cyanogen_msm8916" />
   <project name="LineageOS/android_kernel_cyanogen_msm8974" />
   <project name="LineageOS/android_kernel_cyanogen_msm8994" />
+  <project name="LineageOS/android_kernel_cyanogen_mt6735" />
   <project name="LineageOS/android_kernel_goldfish" />
+  <project name="LineageOS/android_kernel_google_marlin" />
   <project name="LineageOS/android_kernel_google_msm" />
+  <project name="LineageOS/android_kernel_google_seed" />
+  <project name="LineageOS/android_kernel_google_shamrock" />
   <project name="LineageOS/android_kernel_google_steelhead" />
   <project name="LineageOS/android_kernel_hardkernel_odroidc1" />
   <project name="LineageOS/android_kernel_htc_endeavoru" />
@@ -1131,11 +1246,14 @@
   <project name="LineageOS/android_kernel_htc_msm8960" />
   <project name="LineageOS/android_kernel_htc_msm8974" />
   <project name="LineageOS/android_kernel_htc_msm8994" />
+  <project name="LineageOS/android_kernel_htc_msm8996" />
   <project name="LineageOS/android_kernel_huawei_angler" />
   <project name="LineageOS/android_kernel_huawei_kiwi" />
   <project name="LineageOS/android_kernel_huawei_msm8226" />
   <project name="LineageOS/android_kernel_huawei_msm8916" />
   <project name="LineageOS/android_kernel_huawei_msm8928" />
+  <project name="LineageOS/android_kernel_leeco_msm8952" />
+  <project name="LineageOS/android_kernel_leeco_msm8976" />
   <project name="LineageOS/android_kernel_letv_msm8994" />
   <project name="LineageOS/android_kernel_lge_bullhead" />
   <project name="LineageOS/android_kernel_lge_dory" />
@@ -1145,17 +1263,18 @@
   <project name="LineageOS/android_kernel_lge_msm8226" />
   <project name="LineageOS/android_kernel_lge_msm8974" />
   <project name="LineageOS/android_kernel_lge_msm8992" />
+  <project name="LineageOS/android_kernel_lge_msm8996" />
   <project name="LineageOS/android_kernel_lge_pplus" />
   <project name="LineageOS/android_kernel_lge_v4xx" />
   <project name="LineageOS/android_kernel_lge_v500" />
   <project name="LineageOS/android_kernel_lge_v909" />
   <project name="LineageOS/android_kernel_mediatek_sprout" />
-  <project name="LineageOS/android_kernel_moto_shamu" />
   <project name="LineageOS/android_kernel_motorola_apq8084" />
   <project name="LineageOS/android_kernel_motorola_ghost" />
   <project name="LineageOS/android_kernel_motorola_msm8226" />
   <project name="LineageOS/android_kernel_motorola_msm8610" />
   <project name="LineageOS/android_kernel_motorola_msm8916" />
+  <project name="LineageOS/android_kernel_motorola_msm8952" />
   <project name="LineageOS/android_kernel_motorola_msm8960-common" />
   <project name="LineageOS/android_kernel_motorola_msm8960dt-common" />
   <project name="LineageOS/android_kernel_motorola_msm8974" />
@@ -1164,7 +1283,9 @@
   <project name="LineageOS/android_kernel_motorola_omap4-kexec-common" />
   <project name="LineageOS/android_kernel_motorola_otus" />
   <project name="LineageOS/android_kernel_motorola_titan" />
+  <project name="LineageOS/android_kernel_moto_shamu" />
   <project name="LineageOS/android_kernel_nextbit_msm8992" />
+  <project name="LineageOS/android_kernel_nubia_msm8996" />
   <project name="LineageOS/android_kernel_nvidia_roth" />
   <project name="LineageOS/android_kernel_nvidia_shield" />
   <project name="LineageOS/android_kernel_nvidia_shieldtablet" />
@@ -1186,17 +1307,21 @@
   <project name="LineageOS/android_kernel_samsung_espresso10" />
   <project name="LineageOS/android_kernel_samsung_exynos5410" />
   <project name="LineageOS/android_kernel_samsung_exynos5420" />
+  <project name="LineageOS/android_kernel_samsung_exynos7420" />
   <project name="LineageOS/android_kernel_samsung_galaxytab-cdma" />
   <project name="LineageOS/android_kernel_samsung_gts2" />
   <project name="LineageOS/android_kernel_samsung_hlte" />
   <project name="LineageOS/android_kernel_samsung_jf" />
+  <project name="LineageOS/android_kernel_samsung_k3gxx" />
   <project name="LineageOS/android_kernel_samsung_klimtwifi" />
   <project name="LineageOS/android_kernel_samsung_klte" />
   <project name="LineageOS/android_kernel_samsung_ks01lte" />
+  <project name="LineageOS/android_kernel_samsung_kyleproxx" />
   <project name="LineageOS/android_kernel_samsung_lt03lte" />
   <project name="LineageOS/android_kernel_samsung_lt03wifi" />
   <project name="LineageOS/android_kernel_samsung_manta" />
   <project name="LineageOS/android_kernel_samsung_mondrianwifi" />
+  <project name="LineageOS/android_kernel_samsung_ms013g" />
   <project name="LineageOS/android_kernel_samsung_msm8660-common" />
   <project name="LineageOS/android_kernel_samsung_msm8660-q1" />
   <project name="LineageOS/android_kernel_samsung_msm8930-common" />
@@ -1208,12 +1333,15 @@
   <project name="LineageOS/android_kernel_samsung_s3ve3g" />
   <project name="LineageOS/android_kernel_samsung_smdk4210" />
   <project name="LineageOS/android_kernel_samsung_smdk4412" />
+  <project name="LineageOS/android_kernel_samsung_smdk5260" />
   <project name="LineageOS/android_kernel_samsung_t1" />
   <project name="LineageOS/android_kernel_samsung_trelte" />
   <project name="LineageOS/android_kernel_samsung_trlte" />
   <project name="LineageOS/android_kernel_samsung_tuna" />
+  <project name="LineageOS/android_kernel_samsung_universal8890" />
   <project name="LineageOS/android_kernel_samsung_v2wifixx" />
   <project name="LineageOS/android_kernel_samsung_victory" />
+  <project name="LineageOS/android_kernel_samsung_zenltexx" />
   <project name="LineageOS/android_kernel_smartfren_msm8916" />
   <project name="LineageOS/android_kernel_sony_apq8064" />
   <project name="LineageOS/android_kernel_sony_flamingo" />
@@ -1226,21 +1354,27 @@
   <project name="LineageOS/android_kernel_sony_msm8960t" />
   <project name="LineageOS/android_kernel_sony_msm8974" />
   <project name="LineageOS/android_kernel_sony_msm8974pro" />
+  <project name="LineageOS/android_kernel_sony_msm8994" />
   <project name="LineageOS/android_kernel_sony_msm8x27" />
   <project name="LineageOS/android_kernel_sony_msm8x60" />
   <project name="LineageOS/android_kernel_sony_tianchi" />
   <project name="LineageOS/android_kernel_toshiba_betelgeuse" />
   <project name="LineageOS/android_kernel_wingtech_msm8916" />
   <project name="LineageOS/android_kernel_xiaomi_aries" />
+  <project name="LineageOS/android_kernel_xiaomi_armani" />
   <project name="LineageOS/android_kernel_xiaomi_cancro" />
+  <project name="LineageOS/android_kernel_xiaomi_ferrari" />
+  <project name="LineageOS/android_kernel_xiaomi_msm8916" />
   <project name="LineageOS/android_kernel_xiaomi_msm8956" />
+  <project name="LineageOS/android_kernel_xiaomi_msm8996" />
   <project name="LineageOS/android_kernel_yu_msm8916" />
   <project name="LineageOS/android_kernel_zte_msm8994" />
+  <project name="LineageOS/android_kernel_zte_msm8996" />
   <project name="LineageOS/android_libcore" />
   <project name="LineageOS/android_libnativehelper" />
   <project name="LineageOS/android_ndk" />
-  <project name="LineageOS/android_packages_apps_ADWLauncher" />
   <project name="LineageOS/android_packages_apps_AccountsAndSyncSettings" />
+  <project name="LineageOS/android_packages_apps_ADWLauncher" />
   <project name="LineageOS/android_packages_apps_AicBootFix" />
   <project name="LineageOS/android_packages_apps_AndroidTerm" />
   <project name="LineageOS/android_packages_apps_Apollo" />
@@ -1249,6 +1383,13 @@
   <project name="LineageOS/android_packages_apps_Bluetooth" />
   <project name="LineageOS/android_packages_apps_BluetoothExt" />
   <project name="LineageOS/android_packages_apps_Browser" />
+  <project name="LineageOS/android_packages_apps_Calculator" />
+  <project name="LineageOS/android_packages_apps_Calendar" />
+  <project name="LineageOS/android_packages_apps_Camera" />
+  <project name="LineageOS/android_packages_apps_Camera2" />
+  <project name="LineageOS/android_packages_apps_CarrierConfig" />
+  <project name="LineageOS/android_packages_apps_CellBroadcastReceiver" />
+  <project name="LineageOS/android_packages_apps_CertInstaller" />
   <project name="LineageOS/android_packages_apps_CMAccount" />
   <project name="LineageOS/android_packages_apps_CMBugreport" />
   <project name="LineageOS/android_packages_apps_CMFileManager" />
@@ -1261,31 +1402,28 @@
   <project name="LineageOS/android_packages_apps_CMUpdater" />
   <project name="LineageOS/android_packages_apps_CMWallpapers" />
   <project name="LineageOS/android_packages_apps_CMWimaxSettings" />
-  <project name="LineageOS/android_packages_apps_Calculator" />
-  <project name="LineageOS/android_packages_apps_Calendar" />
-  <project name="LineageOS/android_packages_apps_Camera" />
-  <project name="LineageOS/android_packages_apps_Camera2" />
-  <project name="LineageOS/android_packages_apps_CarrierConfig" />
-  <project name="LineageOS/android_packages_apps_CellBroadcastReceiver" />
-  <project name="LineageOS/android_packages_apps_CertInstaller" />
   <project name="LineageOS/android_packages_apps_Contacts" />
   <project name="LineageOS/android_packages_apps_ContactsCommon" />
-  <project name="LineageOS/android_packages_apps_DSPManager" />
+  <project name="LineageOS/android_packages_apps_DesignerTools" />
   <project name="LineageOS/android_packages_apps_DeskClock" />
   <project name="LineageOS/android_packages_apps_Dialer" />
   <project name="LineageOS/android_packages_apps_DockAudio" />
+  <project name="LineageOS/android_packages_apps_DSPManager" />
   <project name="LineageOS/android_packages_apps_Eleven" />
   <project name="LineageOS/android_packages_apps_Email" />
+  <project name="LineageOS/android_packages_apps_EmergencyInfo" />
   <project name="LineageOS/android_packages_apps_ExactCalculator" />
   <project name="LineageOS/android_packages_apps_Exchange" />
-  <project name="LineageOS/android_packages_apps_FM" />
-  <project name="LineageOS/android_packages_apps_FMRadio" />
   <project name="LineageOS/android_packages_apps_FastPowerOn" />
   <project name="LineageOS/android_packages_apps_FileManager" />
+  <project name="LineageOS/android_packages_apps_FlipFlap" />
+  <project name="LineageOS/android_packages_apps_FM" />
+  <project name="LineageOS/android_packages_apps_FMRadio" />
   <project name="LineageOS/android_packages_apps_Focal" />
   <project name="LineageOS/android_packages_apps_Gallery" />
   <project name="LineageOS/android_packages_apps_Gallery2" />
   <project name="LineageOS/android_packages_apps_Gallery3D" />
+  <project name="LineageOS/android_packages_apps_GalleryNext" />
   <project name="LineageOS/android_packages_apps_GanOptimizer" />
   <project name="LineageOS/android_packages_apps_Gello" />
   <project name="LineageOS/android_packages_apps_HTMLViewer" />
@@ -1295,8 +1433,8 @@
   <project name="LineageOS/android_packages_apps_Launcher2" />
   <project name="LineageOS/android_packages_apps_Launcher3" />
   <project name="LineageOS/android_packages_apps_LegacyCamera" />
-  <project name="LineageOS/android_packages_apps_LocPick" />
   <project name="LineageOS/android_packages_apps_LockClock" />
+  <project name="LineageOS/android_packages_apps_LocPick" />
   <project name="LineageOS/android_packages_apps_ManagedProvisioning" />
   <project name="LineageOS/android_packages_apps_Messaging" />
   <project name="LineageOS/android_packages_apps_Mms" />
@@ -1316,8 +1454,11 @@
   <project name="LineageOS/android_packages_apps_Protips" />
   <project name="LineageOS/android_packages_apps_Provision" />
   <project name="LineageOS/android_packages_apps_QuickSearchBox" />
+  <project name="LineageOS/android_packages_apps_Recorder" />
+  <project name="LineageOS/android_packages_apps_SafetyRegulatoryInfo" />
   <project name="LineageOS/android_packages_apps_SamsungServiceMode" />
   <project name="LineageOS/android_packages_apps_Screencast" />
+  <project name="LineageOS/android_packages_apps_SemcKeyboardChanger" />
   <project name="LineageOS/android_packages_apps_Settings" />
   <project name="LineageOS/android_packages_apps_SetupWizard" />
   <project name="LineageOS/android_packages_apps_SmartCardService" />
@@ -1326,14 +1467,16 @@
   <project name="LineageOS/android_packages_apps_SpareParts" />
   <project name="LineageOS/android_packages_apps_SpeechRecorder" />
   <project name="LineageOS/android_packages_apps_Stk" />
+  <project name="LineageOS/android_packages_apps_StorageManager" />
   <project name="LineageOS/android_packages_apps_Superuser" />
-  <project name="LineageOS/android_packages_apps_TSCalibration" />
   <project name="LineageOS/android_packages_apps_Tag" />
   <project name="LineageOS/android_packages_apps_TalkBack" />
   <project name="LineageOS/android_packages_apps_Terminal" />
   <project name="LineageOS/android_packages_apps_ThemeChooser" />
   <project name="LineageOS/android_packages_apps_Torch" />
   <project name="LineageOS/android_packages_apps_Trebuchet" />
+  <project name="LineageOS/android_packages_apps_TSCalibration" />
+  <project name="LineageOS/android_packages_apps_TV" />
   <project name="LineageOS/android_packages_apps_TvSettings" />
   <project name="LineageOS/android_packages_apps_UnifiedEmail" />
   <project name="LineageOS/android_packages_apps_Usb" />
@@ -1348,6 +1491,7 @@
   <project name="LineageOS/android_packages_inputmethods_OpenWnn" />
   <project name="LineageOS/android_packages_inputmethods_PinyinIME" />
   <project name="LineageOS/android_packages_providers_ApplicationsProvider" />
+  <project name="LineageOS/android_packages_providers_BlockedNumberProvider" />
   <project name="LineageOS/android_packages_providers_BookmarkProvider" />
   <project name="LineageOS/android_packages_providers_CalendarProvider" />
   <project name="LineageOS/android_packages_providers_CallLogProvider" />
@@ -1355,6 +1499,7 @@
   <project name="LineageOS/android_packages_providers_DataUsageProvider" />
   <project name="LineageOS/android_packages_providers_DownloadProvider" />
   <project name="LineageOS/android_packages_providers_DrmProvider" />
+  <project name="LineageOS/android_packages_providers_EmergencyInfo" />
   <project name="LineageOS/android_packages_providers_GoogleContactsProvider" />
   <project name="LineageOS/android_packages_providers_MediaProvider" />
   <project name="LineageOS/android_packages_providers_PartnerBookmarksProvider" />
@@ -1363,10 +1508,12 @@
   <project name="LineageOS/android_packages_providers_TvProvider" />
   <project name="LineageOS/android_packages_providers_UserDictionaryProvider" />
   <project name="LineageOS/android_packages_providers_WeatherProvider" />
+  <project name="LineageOS/android_packages_resources_devicesettings" />
   <project name="LineageOS/android_packages_screensavers_Basic" />
   <project name="LineageOS/android_packages_screensavers_PhotoTable" />
   <project name="LineageOS/android_packages_screensavers_WebView" />
   <project name="LineageOS/android_packages_services_AppSuggestService" />
+  <project name="LineageOS/android_packages_services_Car" />
   <project name="LineageOS/android_packages_services_CMAudioService" />
   <project name="LineageOS/android_packages_services_CMHardwareService" />
   <project name="LineageOS/android_packages_services_CMProfileManagerService" />
@@ -1380,6 +1527,7 @@
   <project name="LineageOS/android_packages_services_Telephony" />
   <project name="LineageOS/android_packages_services_ThemeManagerService" />
   <project name="LineageOS/android_packages_services_WeatherService" />
+  <project name="LineageOS/android_packages_themes_HexoLibre" />
   <project name="LineageOS/android_packages_themes_Template" />
   <project name="LineageOS/android_packages_wallpapers_Basic" />
   <project name="LineageOS/android_packages_wallpapers_Galaxy4" />
@@ -1396,8 +1544,10 @@
   <project name="LineageOS/android_prebuilts_gcc_darwin-x86_aarch64_aarch64-linux-android-4.9" />
   <project name="LineageOS/android_prebuilts_gcc_darwin-x86_arm_arm-eabi-4.6" />
   <project name="LineageOS/android_prebuilts_gcc_darwin-x86_arm_arm-linux-androideabi-4.9" />
+  <project name="LineageOS/android_prebuilts_gcc_darwin-x86_x86_x86_64-linux-android-4.9" />
   <project name="LineageOS/android_prebuilts_gcc_linux-x86_aarch64_aarch64-linux-android-4.9" />
   <project name="LineageOS/android_prebuilts_gcc_linux-x86_arm_arm-linux-androideabi-4.9" />
+  <project name="LineageOS/android_prebuilts_gcc_linux-x86_x86_x86_64-linux-android-4.9" />
   <project name="LineageOS/android_prebuilts_misc" />
   <project name="LineageOS/android_prebuilts_ndk" />
   <project name="LineageOS/android_prebuilts_tools" />
@@ -1412,11 +1562,15 @@
   <project name="LineageOS/android_system_netd" />
   <project name="LineageOS/android_system_qcom" />
   <project name="LineageOS/android_system_security" />
+  <project name="LineageOS/android_system_sepolicy" />
   <project name="LineageOS/android_system_su" />
+  <project name="LineageOS/android_system_tools_aidl" />
   <project name="LineageOS/android_system_vold" />
   <project name="LineageOS/android_system_wlan_ti" />
   <project name="LineageOS/android_tools_base" />
+  <project name="LineageOS/android_vendor_apple" />
   <project name="LineageOS/android_vendor_cm" />
+  <project name="LineageOS/android_vendor_codeaurora_telephony" />
   <project name="LineageOS/android_vendor_cyanogen" />
   <project name="LineageOS/android_vendor_gello" />
   <project name="LineageOS/android_vendor_google_passion" />
@@ -1433,20 +1587,23 @@
   <project name="LineageOS/android_vendor_intel_external_openssl_efi_prebuilt" />
   <project name="LineageOS/android_vendor_intel_hardware_realtek_wlan" />
   <project name="LineageOS/android_vendor_intel_hardware_ufo_gen7" />
-  <project name="LineageOS/android_vendor_nxp-nfc_opensource_Nfc" />
   <project name="LineageOS/android_vendor_nxp-nfc_opensource_frameworks" />
   <project name="LineageOS/android_vendor_nxp-nfc_opensource_libnfc-nci" />
+  <project name="LineageOS/android_vendor_nxp-nfc_opensource_Nfc" />
   <project name="LineageOS/android_vendor_qcom_android-open" />
+  <project name="LineageOS/android_vendor_qcom_opensource_bluetooth" />
   <project name="LineageOS/android_vendor_qcom_opensource_data-ipa-cfg-mgr" />
   <project name="LineageOS/android_vendor_qcom_opensource_dataservices" />
   <project name="LineageOS/android_vendor_qcom_opensource_display-frameworks" />
   <project name="LineageOS/android_vendor_qcom_opensource_dpm" />
   <project name="LineageOS/android_vendor_qcom_opensource_kernel-tests_mm-audio" />
+  <project name="LineageOS/android_vendor_qcom_opensource_location" />
   <project name="LineageOS/android_vendor_qcom_opensource_omx_mm-audio" />
   <project name="LineageOS/android_vendor_qcom_opensource_omx_mm-core" />
   <project name="LineageOS/android_vendor_qcom_opensource_omx_mm-video" />
   <project name="LineageOS/android_vendor_qcom_opensource_time-services" />
   <project name="LineageOS/android_vendor_qcom_opensource_v8" />
+  <project name="LineageOS/android_vendor_qcom_opensource_webkit" />
   <project name="LineageOS/android_vendor_qcom_opensource_wlan_prima" />
   <project name="LineageOS/android_vendor_qcom_opensource_wlan_qcacld-2.0" />
   <project name="LineageOS/android_vendor_tmobile_libs_com.tmobile.themes" />
@@ -1454,12 +1611,23 @@
   <project name="LineageOS/android_vendor_tmobile_providers_ThemeManager" />
   <project name="LineageOS/android_vendor_tmobile_themes_Androidian" />
   <project name="LineageOS/android_vendor_tmobile_themes_Cyanbread" />
-  <project name="LineageOS/cm-kernel" />
+  <project name="LineageOS/AR6kSDK.3.1" />
+  <project name="LineageOS/bugproxy" />
+  <project name="LineageOS/CMAccountWebsite" />
+  <project name="LineageOS/CMBalance" />
   <project name="LineageOS/cm_crowdin" />
+  <project name="LineageOS/cm-kernel" />
+  <project name="LineageOS/cm-less" />
   <project name="LineageOS/cm_platform_sdk" />
+  <project name="LineageOS/CMStatsServer" />
   <project name="LineageOS/ctso_supplicant" />
+  <project name="LineageOS/cve_tracker" />
+  <project name="LineageOS/device_htc_marvel" />
+  <project name="LineageOS/device_htc_ville" />
+  <project name="LineageOS/docker_build" />
   <project name="LineageOS/external_android_support-prebuilt" />
   <project name="LineageOS/external_cyanogen_cmhomeapi" />
+  <project name="LineageOS/geeksphone-kernel-zero" />
   <project name="LineageOS/getcm-translations" />
   <project name="LineageOS/hardkernel-kernel-4412" />
   <project name="LineageOS/hp-kernel-tenderloin" />
@@ -1469,14 +1637,19 @@
   <project name="LineageOS/htc-kernel-liberty" />
   <project name="LineageOS/htc-kernel-msm7227" />
   <project name="LineageOS/htc-kernel-msm7x30" />
+  <project name="LineageOS/htc-kernel-msm8660" />
+  <project name="LineageOS/htc-kernel-pyramid" />
   <project name="LineageOS/htc-kernel-supersonic" />
   <project name="LineageOS/huaweigeneric-ril" />
   <project name="LineageOS/hudson" />
   <project name="LineageOS/ion" />
+  <project name="LineageOS/jenkins_build" />
   <project name="LineageOS/kernel-omap" />
+  <project name="LineageOS/lge-kernel-e400" />
   <project name="LineageOS/lge-kernel-gproj" />
   <project name="LineageOS/lge-kernel-iproj" />
   <project name="LineageOS/lge-kernel-mako" />
+  <project name="LineageOS/lge-kernel-msm7x27" />
   <project name="LineageOS/lge-kernel-msm7x30" />
   <project name="LineageOS/lge-kernel-omap4" />
   <project name="LineageOS/lge-kernel-p700" />
@@ -1484,7 +1657,11 @@
   <project name="LineageOS/lge-kernel-sniper" />
   <project name="LineageOS/lge-kernel-star" />
   <project name="LineageOS/libbt-hci" />
+  <project name="LineageOS/lineageos_updater" />
+  <project name="LineageOS/mirror" />
+  <project name="LineageOS/mirrorbits" />
   <project name="LineageOS/motorola-kernel-stingray" />
+  <project name="LineageOS/samsung-kernel-c1" />
   <project name="LineageOS/samsung-kernel-galaxys" />
   <project name="LineageOS/samsung-kernel-hercules" />
   <project name="LineageOS/semc-kernel-msm7x30" />
@@ -1492,8 +1669,12 @@
   <project name="LineageOS/sony-kernel-msm8660" />
   <project name="LineageOS/sony-kernel-msm8960" />
   <project name="LineageOS/sony-kernel-u8500" />
+  <project name="LineageOS/Superuser" />
+  <project name="LineageOS/temp_infra_tracker" />
   <project name="LineageOS/themes-platform-vendor-tmobile-apps-ThemeChooser" />
   <project name="LineageOS/tools_repo" />
+  <project name="LineageOS/tribble-tracker" />
+  <project name="LineageOS/Widgets" />
+  <project name="LineageOS/www" />
   <project name="LineageOS/zte-kernel-msm7x27" />
 </manifest>
-
-- 
2.17.1

